packages/server/src/integrations/microsoftSqlServer.ts

```diff
@@ -166,7 +166,7 @@ const SCHEMA: Integration = {
       fields: {
         domain: {
           type: DatasourceFieldType.STRING,
-          required: false,
```

Comment:
Will this leave some customers in a sticky situation if they've saved their config when this wasn't required? Or are those users already broken?

Reply:
No. This is only UI, but the actual implementation requires it.
![image](https://github.com/user-attachments/assets/f7b11404-3eeb-4981-bb09-e617f03b327d)

This will better manage the error on the frontend